env.example

```diff
@@ -12,4 +12,5 @@ IPFS_API_KEY=
 IPFS_API_SECRET=
 SLACK_REPORTING_URL=
 WEB3_STORAGE_TOKEN=
+# To be able to access dashboard http://<serverIP>:3000/ remotelly, Add server IP in format below in format POOLER_API_PREFIX=http://<serverIP>:8002
```

Comment:
Let's change this to:

`# To be able to access Pooler frontend  (http://<serverIP>:3000) on a remote instance, add the server IP in format below like this: POOLER_API_PREFIX=http://<serverIP>:8002`